src/content/docs/mobile-monitoring/new-relic-mobile-ios/installation/spm-installation.mdx

```diff
@@ -46,16 +46,15 @@ Continue with the steps to configure New Relic for mobile monitoring.
    </Callout>
 
 5. Add a build script to your target's **Build Phases**. Ensure the new build script is the very last build script. Then paste the following, replacing `APP_TOKEN` with your [application token](/docs/mobile-apps/viewing-your-application-token):
-    - ###  **iOS Agent 7.4.0+:**
+    * iOS agent 7.4.0 or higher:
```

Comment:
We want to make sure the reader understands that these are separate instructions and not listed instructions. I recommend keeping these bigger and bold. Do this for the 7.3.8 instructions and on the other files that contain these instructions.
```suggestion
    ### ** iOS agent 7.4.0 or higher:**
```